src/main/webapp/js/restaurant-page.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Configures restaurant name and photo.
+ * @param {Object} restaurant restaurant object
```

Comment:
```suggestion
 * @param {Object} restaurant object
```

Reply:
the original looks correct, the param is `restaurant`, and the description is `restaurant object`